lib/Github/Api/AbstractApi.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
I don't think this is in the way of think of the project. Keeping the PHP way looks like a better alternative. Also, the indentation is wrong (it should definitely be on a new line)